fix_hmc.txt

```diff
@@ -0,0 +1,46 @@
+"LAMMPS WWW Site"_lws - "LAMMPS Documentation"_ld - "LAMMPS Commands"_lc :c
+
+:link(lws,http://lammps.sandia.gov)
+:link(ld,Manual.html)
+:link(lc,Section_commands.html#comm)
+
+:line
+
+fix hmc command :h3
+
+[Syntax:]
+
+fix ID group-ID hmc N integrator seed T keyword values ... :pre
+
+ID, group-ID are documented in "fix"_fix.html command :ulb,l
+hmc = style name of this fix command :l
+N = run MD N steps between MC acceptance protocol :l
```

Comment:
We could add run **NVE** MD to emphasize that molecular dynamics steps correspond to the microcanonical ensemble